### ISSUE TITLE
Quote lines before Markdown in Python egg PKGINFO files cause spectrometer to error out

### ISSUE THREAD

**Issue by aaron-p-lehmann** [fossas/spectrometer#229]
When the PKGINFO of a Python 2.7 egg has a “=====” line in its markup, the command `fossa container analyze <image that contains the egg>` will fail with an error like the following:
```
----------
A fatal error occurred:
 
    Command execution failed: CmdFailure {cmdFailureName = "/sysroot/tmp/fossa-vendor/syft", cmdFailureArgs = ["-o","json","integrations/integrations"], cmdFailureDir = "/var/roothome/fossa/", cmdFailureExit = ExitFailure 1, cmdFailureStdout = "", cmdFailureStderr = "\ESC[90m[0024]\ESC[0m \ESC[31mERROR\ESC[0m failed to catalog input: 1 error occurred:\n\t* unable to catalog python package=/usr/lib/python2.7/site-packages/cabby-0.1.20-py2.7.egg-info
/PKG-INFO: cannot parse field from line: '======='\n\n\n"}
      
 
>>> 
Relevant warnings include:
```
 
Removing the ```        =====``` in the PKGINFO will enable the analysis to progress, but that’s obviously not the right way to do things.
 
PKGINFO that causes the error (from cabby-0.1.20):
```
Metadata-Version: 1.1
Name: cabby
Version: 0.1.20
Summary: TAXII client library
Home-page: https://github.com/EclecticIQ/cabby/
Author: EclecticIQ
Author-email: cabby@eclecticiq.com
License: BSD License
Description: cabby
        =====
        
        Python TAXII client implementation from `EclecticIQ <https://www.eclecticiq.com>`_.
        
        :Source: https://github.com/EclecticIQ/cabby
        :Documentation: http://cabby.readthedocs.org
        :Information: https://www.eclecticiq.com
        :Download: https://pypi.python.org/pypi/cabby/
        
        |travis badge| |landscape.io badge| |coveralls.io badge| |docs badge| |requirements badge|
        
        .. |travis badge| image:: https://travis-ci.org/EclecticIQ/cabby.svg?branch=master
           :target: https://travis-ci.org/EclecticIQ/cabby
           :alt: Build Status
        .. |landscape.io badge| image:: https://landscape.io/github/EclecticIQ/cabby/master/landscape.svg?style=flat
           :target: https://landscape.io/github/EclecticIQ/cabby/master
           :alt: Code Health
        .. |coveralls.io badge| image:: https://coveralls.io/repos/EclecticIQ/cabby/badge.svg
           :target: https://coveralls.io/r/EclecticIQ/cabby
           :alt: Coverage Status
        .. |docs badge| image:: https://readthedocs.org/projects/cabby/badge/?version=latest
            :alt: Documentation Status
            :scale: 100%
            :target: https://readthedocs.org/projects/cabby/
        .. |requirements badge| image:: https://requires.io/github/EclecticIQ/cabby/requirements.svg?branch=master
             :target: https://requires.io/github/EclecticIQ/cabby/requirements/?branch=master
             :alt: Requirements Status
        
        A simple Python library for interacting with TAXII servers.
        
        
        Docker
        --------
        
        From version 0.1.13, the docker image is based on 'Alpine' linux. This means the size of the Image was reduced from 311MB to 74MB
        
        To run cabby using docker, execute the following:
        
          docker run --rm=true eclecticiq/cabby:latest taxii-discovery --path https://test.taxiistand.com/read-only/services/discovery
        
        Feedback
        --------
        
        You are encouraged to provide feedback by commenting on open issues or sending us 
        email at cabby@eclecticiq.com
```

Removing the ```        =====``` will fix the problem:

```
Metadata-Version: 1.1
Name: cabby
Version: 0.1.20
Summary: TAXII client library
Home-page: https://github.com/EclecticIQ/cabby/
Author: EclecticIQ
Author-email: cabby@eclecticiq.com
License: BSD License
Description: cabby
        
        Python TAXII client implementation from `EclecticIQ <https://www.eclecticiq.com>`_.
        
        :Source: https://github.com/EclecticIQ/cabby
        :Documentation: http://cabby.readthedocs.org
        :Information: https://www.eclecticiq.com
        :Download: https://pypi.python.org/pypi/cabby/
        
        |travis badge| |landscape.io badge| |coveralls.io badge| |docs badge| |requirements badge|
        
        .. |travis badge| image:: https://travis-ci.org/EclecticIQ/cabby.svg?branch=master
           :target: https://travis-ci.org/EclecticIQ/cabby
           :alt: Build Status
        .. |landscape.io badge| image:: https://landscape.io/github/EclecticIQ/cabby/master/landscape.svg?style=flat
           :target: https://landscape.io/github/EclecticIQ/cabby/master
           :alt: Code Health
        .. |coveralls.io badge| image:: https://coveralls.io/repos/EclecticIQ/cabby/badge.svg
           :target: https://coveralls.io/r/EclecticIQ/cabby
           :alt: Coverage Status
        .. |docs badge| image:: https://readthedocs.org/projects/cabby/badge/?version=latest
            :alt: Documentation Status
            :scale: 100%
            :target: https://readthedocs.org/projects/cabby/
        .. |requirements badge| image:: https://requires.io/github/EclecticIQ/cabby/requirements.svg?branch=master
             :target: https://requires.io/github/EclecticIQ/cabby/requirements/?branch=master
             :alt: Requirements Status
        
        A simple Python library for interacting with TAXII servers.
        
        
        Docker
        --------
        
        From version 0.1.13, the docker image is based on 'Alpine' linux. This means the size of the Image was reduced from 311MB to 74MB
        
        To run cabby using docker, execute the following:
        
          docker run --rm=true eclecticiq/cabby:latest taxii-discovery --path https://test.taxiistand.com/read-only/services/discovery
        
        Feedback
        --------
        
        You are encouraged to provide feedback by commenting on open issues or sending us 
        email at cabby@eclecticiq.com
        
        
Platform: UNKNOWN
Classifier: Development Status :: 4 - Beta
Classifier: Intended Audience :: Developers
Classifier: Intended Audience :: Information Technology
Classifier: License :: OSI Approved :: BSD License
Classifier: Programming Language :: Python
Classifier: Programming Language :: Python :: 2.7
Classifier: Programming Language :: Python :: 3.4
Classifier: Topic :: Internet
Classifier: Topic :: Security
Classifier: Topic :: Software Development :: Libraries :: Python Modules
```

**Comment by enricozb** [comment]
I am unable to reproduce this with an image that includes cabby. It's extremely peculiar that removing `=====` from the cabby `PKGINFO` fixes this, because the `=======` from the error message has a different number of `=` signs. However, the `LICENSE.rst` from cabby has the exact number of `=` from the error message. This makes me guess that somewhere in some `setup.py`, some code like this exists:
```
setup(
    ...
    license=open("LICENSE.rst").read(),
    ...
)
```
which is incorrect because `license` is a short string as per [these docs](https://docs.python.org/3/distutils/setupscript.html#additional-meta-data).

Is there any chance you can provide the dockerfile that is used to build the image that is failing? Here is what I tried to use to reproduce:
```
FROM python:3.9-slim AS base

ARG PIP_NO_CACHE_DIR=1

RUN pip install -U pip
RUN pip install cabby==0.1.20

WORKDIR /app
RUN useradd -m -u 777 mrdocker && chown mrdocker:mrdocker /app
FROM base AS runtime

USER mrdocker

LABEL maintainer="Security"

ENTRYPOINT ["python3", "main.py"]
```

**Comment by Serubin** [comment]
This issue is resolved with the latest release [v2.5.11](https://github.com/fossas/spectrometer/releases/tag/v2.5.11)
